src/main/java/com/github/arkadiusz97/sms_filter_service/smsfilterservice/sms_operator_service/service/SmsService.java

```diff
@@ -0,0 +1,113 @@
+package com.github.arkadiusz97.sms_filter_service.smsfilterservice.sms_operator_service.service;
+
+import com.github.arkadiusz97.sms_filter_service.smsfilterservice.domain.ConfidenceLevel;
+import com.github.arkadiusz97.sms_filter_service.smsfilterservice.domain.PhoneNumberRequest;
+import com.github.arkadiusz97.sms_filter_service.smsfilterservice.domain.PhoneNumberResponse;
+import com.github.arkadiusz97.sms_filter_service.smsfilterservice.sms_operator_service.domain.Client;
+import com.github.arkadiusz97.sms_filter_service.smsfilterservice.sms_operator_service.domain.Sms;
+import com.github.arkadiusz97.sms_filter_service.smsfilterservice.sms_operator_service.domain.SmsServiceResponse;
+import com.github.arkadiusz97.sms_filter_service.smsfilterservice.sms_operator_service.domain.SmsServiceStatus;
+import com.github.arkadiusz97.sms_filter_service.smsfilterservice.sms_operator_service.repository.ClientRepository;
+import okhttp3.OkHttpClient;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.stereotype.Service;
+import retrofit2.Call;
+import retrofit2.Response;
+import retrofit2.Retrofit;
+import retrofit2.converter.gson.GsonConverterFactory;
+
+import java.io.FileInputStream;
+import java.util.Properties;
+
+@Service
+public class SmsService {
+
+    private OkHttpClient.Builder httpClient;
+    private Retrofit retrofit;
+    private EvaluateSmsService evaluateSmsService;
+    private String operatorSpecialNumber = "special-number";
+    public SmsService() {
+        httpClient = new OkHttpClient.Builder();
+        retrofit = new Retrofit.Builder()

```

Comment:
Czy retrofit w tej aplikacji jest użyty by stworzyć typesafe call'a do samej siebie by przykryć brak faktycznej implementacji? :fearful: 
Co by nie było brawo za pomysłowość :sweat_smile:

Reply:
Główna moja motywacja do użycia retrofita to zapewnienie czystego i czytelnego kodu. Poza tym tę bibliotekę znam z projektu, który rozwijam w swojej obecnej pracy. Typesafe call to też duży plus, zawsze mam pewność, że odpowiedź od serwera będzie mieć zawsze taką samą strukturę.